pheme/transformation/scanreport/gvmd.py

```diff
@@ -196,12 +198,14 @@ def __tansform_tags(item) -> List[Dict]:
     return None
 
 
-def __return_highest_threat(old: str, new: str) -> str:
-    if old == 'High' or new == 'High':
-        return 'High'
-    if old == 'Medium' or new == 'Medium':
-        return 'Medium'
-    return 'Low'
+def __return_highest_threat(threats: List[int]) -> str:
+    """
+    retuns the highest threat within a list of threats based on __threats
+    """
+    for i, value in enumerate(threats):
+        if value > 0:
```

Comment:
Isn't this returning the first value > 0 and not the highest value?
Enumerate is not sorting the values, does it?

Reply:
It is not about the highest value but the highest threat and the index of __threats is sorted.